src/utils/fetch-config.ts

```diff
@@ -1,110 +1,18 @@
 /* eslint-disable no-console */
 
-import axios from '@/axios-auth'
-
 /**
- * Fetches config from environment and API.
- * Also identifies Business ID from initial route.
+ * stores BUSINESS_ID, TEMP_REG_NUMBER, BASE_URL, VUE_ROUTER_BASE from initial route.
  */
-export async function fetchConfig (): Promise<void> {
-  // get config from environment
-  const processEnvBaseUrl: string = process.env.BASE_URL // eg, /business/
-  const windowLocationPathname = window.location.pathname // eg, /business/CP1234567/...
-  const windowLocationOrigin = window.location.origin // eg, http://localhost:8080
 
-  if (!processEnvBaseUrl || !windowLocationPathname || !windowLocationOrigin) {
+export function setBaseRouteAndBusinessId (pathname: string, processEnvBaseUrl: string, rootUrl: string): Promise<any> {
+  if (!processEnvBaseUrl || !pathname || !rootUrl) {
```

Comment:
Given how this file had changed so significantly, I propose renaming it.

Reply:
Good idea.  I'll rename it.